cider/util/StringTransform.cpp

```diff
@@ -68,7 +66,7 @@ std::vector<std::pair<size_t, size_t>> find_string_literals(const std::string& q
       if (!boost::regex_search(it, query.end(), what, literal_string_regex)) {
         break;
       }
-    } catch (const std::exception& e) {
+    } catch (const CiderException& e) {
```

Comment:
should not change here. boost lib will never throw CiderException.

Reply:
fix